### ISSUE TITLE
Play sound upon completion of build so that users can multi-task (do work in other applications) and know when the build has completed

### ISSUE THREAD

**Issue by dabelt-msft** [microsoft/vscode#141506]
I see others have requested the ability to play a sound upon the completion of a build. This request was duplicated seemingly erroneously here (https://github.com/microsoft/vscode/issues/12466) because visual indication of build status was provided in the terminal; but visual indication requires that people stick around to watch the build or check back frequently, breaking concentration. Enabling a user to toggle an audible indication of a build failure or completion increases the ability to multitask.

**Comment by diminutivesloop** [comment]
Ideally this would also support visual notification such as OS notifications for situations where an audible notification won't be heard such as deafness or muted speakers.

**Comment by isidorn** [comment]
Our sound designer create a sound for this, so we can start on the implementation in September.
In addition to the completion sound, the designer created a sound for when the task started. I am not 100% convinced we need this sound. So I suggest we start just with the "task finished" sound. But I am open to suggestions.

**Comment by dabelt-msft** [comment]
Regarding the sound to indicate the process started - I agree that it doesn't seem necessary, and I would go further to say that some people might dislike it. If implemented would be good to have an option to toggle both independently because my guess is most people would only want the completion sound. Otherwise it could be annoying to have too many sounds, some of which are unnecessary.

**Comment by isidorn** [comment]
@dabelt-msft I agree.
@meganrogge maybe we start with just a "task finished" sound. Do let me know if I can help here. Thanks!

**Comment by bmartin** [comment]
Personally I'd love a little victory trill (do Do DOO) on zero exit code
and a sad trombone (wah-wah) noise on non-zero. To be really greedy, I'd
pick different noises for low exits like 1 or 2 and high ones like 255. I
have at least one colleague who would probably pick a football theme
"touchdown"/"fumble"/"interception".  Could become a beloved mini feature.
